apps/system/test/unit/statusbar_test.js

```diff
@@ -876,6 +876,7 @@ suite('system/Statusbar', function() {
     });
 
     test('sheets-gesture-end', function() {
+      StatusBar._pausedForGesture = true;
```

Comment:
please do not set internal variables from tests, keep the tests "externals" you probably want to dispatch a `sheets-gesture-begin` instead.

It keeps the tests more robust (ie. won't have to change the test if we change the internal variable name) and closer to what really happens.